bundles/framework/divmanazer/component/VisualizationForm.js

```diff
@@ -126,24 +149,25 @@ Oskari.clazz.define(
 
         /**
          * @public @method getValues
-         * Returns the values of each form clazz.
+         * Returns the values of each form clazz and optional name of style
          *
          *
          * @return {Object}
          */
         getValues: function () {
-            var values = {},
-                formClazzes = this._getFormClazz(),
-                fClazzName,
-                fClazz;
+            const values = {};
+            const formClazzes = this._getFormClazz();
 
-            for (fClazzName in formClazzes) {
+            for (let fClazzName in formClazzes) {
                 if (formClazzes.hasOwnProperty(fClazzName)) {
-                    fClazz = formClazzes[fClazzName];
+                    let fClazz = formClazzes[fClazzName];
                     values[fClazzName] = fClazz.getValues();
                 }
             }
-
+            const styleNameInputValue = jQuery('.wfs_own_style').find('input[name="styleNameInput"]').val();
```

Comment:
Don't use '.wfs_own_style' selector. It's added by `MapWfs2` bundle and can't be relied to. Is there a way pointing to the dialog directly without searching from the document?

Reply:
Done by storing created input field as a property of VisualizationForm and reading value from it.